### PR TITLE
AST-2207 - Gradient angle input field not visible in the block editor

### PR DESCRIPTION
**Pull request by GauravKhupse**
### Description

The gradient angle input field is not well visible in the Customizer and Gutenberg editor. It’s somehow 2 digits visible in Gutenberg, but only 1 digit is visible when you are on Customizer.

In the Gutenberg editor: [Annotation on 2022-09-12 at 18-58-04.png](https://d.pr/i/l48OzA) 

 

An angle can be a maximum of 3 digits; the whole 3 digits should be visible.
### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: sass/admin/wp-editor-styles.scss [hunk 1]
```diff
@@ -466,4 +466,16 @@ pre.wp-block {
...
\ No newline at end of file
+}
+
+// Added css for gradirnt color picker
+.block-editor-page {
+	.components-angle-picker-control {
+		.components-input-control__container {
+			.components-input-control__input {
+				padding-left: calc(10px);
+				padding-right: calc(10px);
```
Comment (imnavanath) [inline sass/admin/wp-editor-styles.scss:477]:
```suggestion
				padding-left: 10px;
				padding-right: 10px;
```

File: sass/admin/wp-editor-styles.scss [hunk 2]
```diff
@@ -466,4 +466,16 @@ pre.wp-block {
...
 		}
     }
-}
\ No newline at end of file
+}
+
+// Added css for gradirnt color picker
+.block-editor-page {
+	.components-angle-picker-control {
+		.components-input-control__container {
```
Comment (imnavanath) [inline sass/admin/wp-editor-styles.scss:474]: Can this be selector reduce? basically want lower CSS weightage, so new can be - 
```
.block-editor-page {
	.components-angle-picker-control {
		.components-input-control__input {
```
Please check once.

File: changelog.txt [hunk 3]
```diff
@@ -10,6 +10,7 @@ v4.0.0 (Unreleased)
 - Fix: Block editor - Getting "wp.data.select( 'core/editor' )getBlocks()" console warning in editor as being deprecated in WordPress version 5.3.
 - Fix: Block editor - Button background colour gets overridden by core styles for Legacy block editor setup.
 - Fix: Getting console error in customizer when trying to update footer columns.
+- Fix: Gradient angle input field not visible in the block editor.
```
Comment (imnavanath) [inline changelog.txt:13]:
```suggestion
- Fix: Block editor - Color gradient angle input field not visible in the block settings.
```

File: sass/admin/wp-editor-styles.scss (hunk 2 above)
Comment (GauravKhupse) [reply sass/admin/wp-editor-styles.scss:474]: I checked after reducing CSS, and the issue still persists. so it needed

File: changelog.txt (hunk 3 above)
Comment (GauravKhupse) [reply changelog.txt]: Changes done!

File: sass/admin/wp-editor-styles.scss (hunk 1 above)
Comment (GauravKhupse) [reply sass/admin/wp-editor-styles.scss]: Changes done!

**Review by imnavanath — APPROVED** [review]

**Review by yssambare12 — APPROVED** [review]

**Pull request merged**